### DISCUSSION TITLE
Interests icon is missing when viewing profile in-game

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#26258] [Q&A] [answered]
What it looks like on the website:

![image](https://github.com/ppy/osu/assets/74088315/20f99592-6916-4ddc-8249-21d40727bb1f)

What it looks like in-game (missing heart icon):

![image](https://github.com/ppy/osu/assets/74088315/920f76df-d037-4170-8270-d4429cdf1874)

**Comment by Gabixel** [comment, marked as answer]
Fixed with #26276
